src/cdk/schematics/ng-generate/drag-drop/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.html

```diff
@@ -0,0 +1,15 @@
+<div class="container">
+  <h2>To do</h2>
+
+  <cdk-drop #todoList [data]="todo" [connectedTo]="[doneList]" class="list" (dropped)="drop($event)">
```

Comment:
The `connectedTo` accepts a single reference as well: `[connectedTo]="doneList"`.

Reply:
I've basically taken this from the example on `material.angular.io`. What's your preference?

Reply:
I'd say passing in the single one since it's a bit cleaner.